Gemfile

```diff
@@ -46,3 +46,5 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+#Install react for rails
+gem 'react-rails'
```

Comment:
Not related to this project, but you maybe want to took a look at `.editorconfig` and `.eslintrc` files from referrals project. First one is supported by RubyMine natively, second requires plugin.

They help to use same code style for all code collaborators.